README.rst

```diff
@@ -778,9 +778,7 @@ Linux with bonded interfaces and disabled NetworkManager
             address: 192.168.0.102
             netmask: 255.255.255.0
             mtu: 1500
-            use_in:
-            - interface: ${linux:interface:eth0}
-            - interface: ${linux:interface:eth0}
+            slaves: eth0 eth1
```

Comment:
change to list

Reply:
I'll see how i can make that backwards compatible then, because i updated the readme to reflect how the salt-formula works